spec.html

```diff
@@ -36247,16 +36247,24 @@ <h1>
           <h1>
             GetStringIndex (
               _S_: a String,
-              _e_: a non-negative integer,
+              _cpIndex_: a non-negative integer,
             ): a non-negative integer
           </h1>
           <dl class="header">
+            <dt>description</dt>
+            <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the code unit index corresponding with code point index _cpIndex_, clipped to a maximum position just beyond the end of _S_.</dd>
```

Comment:
Nit:
```suggestion
            <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the code unit index corresponding to code point index _cpIndex_, clamped to a maximum position just beyond the end of _S_.</dd>
```

The `with` -> `to` just as a matter of grammar, the `clipped` -> `clamped` because [that is the term we use in the spec](https://tc39.es/ecma262/multipage/notational-conventions.html#clamping).